goldens/public-api/forms/forms.d.ts

```diff
@@ -349,7 +349,7 @@ export declare const NG_ASYNC_VALIDATORS: InjectionToken<(Function | Validator)[
 
 export declare const NG_VALIDATORS: InjectionToken<(Function | Validator)[]>;
 
-export declare const NG_VALUE_ACCESSOR: InjectionToken<ControlValueAccessor>;
+export declare const NG_VALUE_ACCESSOR: InjectionToken<readonly ControlValueAccessor[]>;
```

Comment:
Hi @Airblader,

Sorry I didn't mention that yesterday: I think the implementation that you have would work, but we usually have the `ReadonlyArray` type in these cases, i.e.:

```suggestion
export declare const NG_VALUE_ACCESSOR: InjectionToken<ReadonlyArray<ControlValueAccessor>>;
```

Could you please update the PR to use `ReadonlyArray` instead?

Thank you.